app/helpers/date_time_helper.rb

```diff
@@ -0,0 +1,9 @@
+# frozen_string_literal: true
+
+module DateTimeHelper
+  def date_time(date)
+    return date.strftime('%H:%M') if (Time.zone.now - 1.day) < date
+
+    date.strftime('%F %H:%M')
```

Comment:
There already are time formats in the [German locale](
https://github.com/tactilenews/100eyes/blob/master/config/locales/de.yml#L280). All you need to do is add a new date formats for the short and long version, then you can use the i18n helpers:

```ruby
I18n.l date, format: :time_only # no idea what would be an appropriate name...
I18n.l date, fromat: :short_date_time
```ruby